pkg/kubelet/kubelet_node_status.go

```diff
@@ -166,9 +166,10 @@ func (kl *Kubelet) initialNode() (*api.Node, error) {
 		ObjectMeta: api.ObjectMeta{
 			Name: kl.nodeName,
 			Labels: map[string]string{
-				unversioned.LabelHostname: kl.hostname,
-				unversioned.LabelOS:       goRuntime.GOOS,
-				unversioned.LabelArch:     goRuntime.GOARCH,
+				unversioned.LabelHostname:       kl.hostname,
+				unversioned.LabelOS:             goRuntime.GOOS,
+				unversioned.LabelArch:           goRuntime.GOARCH,
+				unversioned.LabelFluentdDsReady: "true",
```

Comment:
IIUC this is temporary? After how many versions do you think it will be okay to drop this label?